packages/embark/src/lib/modules/blockchain_process/parityClient.js

```diff
@@ -221,13 +235,16 @@ class ParityClient {
       cmd.push("--ws-port=" + config.wsPort);
       cmd.push("--ws-interface=" + (config.wsHost === 'localhost' ? 'local' : config.wsHost));
       if (config.wsOrigins) {
-        if (config.wsOrigins === '*') {
+        const origins = config.wsOrigins.split(',');
+        if (origins.includes('*')) {
```

Comment:
We actually need to check for `all` here as well. `*` seems to be a syntax provided by Embark, not Parity.

So theoretically, once could pass in `all,foo,bar`, in which case we'll end up in the `else` branch of this condition, resulting in: `--ws-origins=all,foo.bar` and then we have the problem again that you're trying to fix.

Reply:
I'd like to keep `*`, as it is the CORS specific nomenclature (is this the right word?) I agree tho, that we should pass `all` to Parity. After all, the responsibility for this class is to translate config into Parity parameters.

Reply:
Thanks for updating the PR @andremedeiros !

Yes, Embark has to take care of converting `*` to `all`. However, we still need to **also** check for `all` in the `.includes()` condition. Otherwise:

> So theoretically, one could pass in all,foo,bar, in which case we'll end up in the else branch of this condition, resulting in: --ws-origins=all,foo.bar and then we have the problem again that you're trying to fix.

Reply:
i think this has been fixed, no?

Reply:
Nope. Line `239` still only checks for `*`